tools/flang2/flang2exe/llutil.cpp

```diff
@@ -54,6 +54,54 @@ typedef struct LLDEF {
   struct LLDEF *next;
 } LLDEF;
 
+static const char *openmp_functions[] = {
```

Comment:
Do you want to mark this as AOCC begin/end as well? (since you are doing that below)